Enterprise-CMCS/macpro-mdct-hcbs
# Do some refactoring in the API

### Description
* Use the API response helper pattern from MFP
* Remove prototype HelloWorld endpoint
* Small refactor in buildReport
* Add strongly-typed path parameter validation
* Remove duplicate debug-lib
* Remove default export from dynamodb-lib
* Rename isAuthorized -> isAuthenticated
* Move SSM access to a storage helper

### Related ticket(s)
n/a

---
### How to test
There's not all that much to test here. If the UI can still talk to the API, to create reports and fetch them, then I think we're good.
1. Log in to the app at https://dcy3rnsfblq1k.cloudfront.net/ with `stateuser@test.com`
2. Click through to "enter HCBS QM online"
3. Click "Generate Form" to Create a report
4. You should land on the report page, with all its placeholder glory.

### Important updates
n/a

---
### Author checklist
